spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -46,6 +46,22 @@ ported over by exchanging the package names.
 
 It will be possible to use the non-annotations API, but using the annotations will generally be easier for developers.
 
+[[app-multi-app]]
+=== Multi-Application support
+
+It is expected in a micro services environment that a server hosts one application only.
+This assumption is not always true.
+MicroProfile Metrics supports multiple applications deployed in one server.
+The default is the one application per server case.
+
+MicroProfile metrics will keep per application registries when more than application is deployed.
+
+The name of the application is determined by its `@javax.ws.rs.ApplicationPath` value.
```

Comment:
Should we always strip the leading slash from the path? How about the `/` path?
In the latter case it would be the empty string, which is ok for the internal model, but perhaps needs special consideration on exporting as label